utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -1011,47 +1050,48 @@ inline void compare_output(T* output, string funcName, RpptDescPtr srcDescPtr, R
                     func += "Tensor_PKD3";
                 else
                     func += "Tensor_PLN3";
+                pln1RefStride = 0;
             }
             else
                 func += "Tensor_PLN1";
         }
     }
     if(testCase == 21 ||testCase == 23 || testCase == 24)
+    {
         func += "_interpolationType" + interpolationTypeName;
+        binFile += "_interpolationType" + interpolationTypeName;
+    }
     else if(testCase == 8)
+    {
         func += "_noiseType" + noiseTypeName;
-    refFile = scriptPath + "/../REFERENCE_OUTPUT/" + funcName + "/"+ func + ".csv";
-
-    ifstream file(refFile);
-    Rpp8u *refOutput;
-    refOutput = (Rpp8u *)malloc(dstDescPtr->n * refOutputSize * sizeof(Rpp8u));
+        binFile += "_noiseType" + noiseTypeName;
+    }
+    refFile = scriptPath + "/../REFERENCE_OUTPUT/" + funcName + "/"+ binFile + ".bin";
     string line,word;
     int index = 0;
 
-    // Load the refennce output values from files and store in vector
-    if(file.is_open())
-    {
-        while(getline(file, line))
-        {
-            stringstream str(line);
-            while(getline(str, word, ','))
-            {
-                refOutput[index] = stoi(word);
-                index++;
-            }
-        }
-    }
-    else
-    {
-        cout<<"Could not open the reference output. Please check the path specified\n";
-        return;
-    }
+    FILE *fp;
+    fp = fopen(refFile.c_str(), "rb");
+    if (fp == NULL)
+        printf("Error opening file");
+
+    fseek(fp, 0, SEEK_END);
+    long fsize = ftell(fp);
+    if (fsize == 0)
+        std::cerr << "File is empty";
+
+    fseek(fp, 0, SEEK_SET);
+    Rpp8u *binaryContent = (Rpp8u *)malloc(fsize);
```

Comment:
We can try doing a static allocation as abishek said, but read the data
which is of size fsize only

Instead of dynamically allocating for every variant, better to static allocate as abishek said

Reply:
Done